src/tcpstream.c

```diff
@@ -6,16 +6,18 @@
 
 #include "tcpstream.h"
 
+#define MAX_IO_PER_POLL 32
+
 /************************************************************/
 //		ntttcp socket functions
 /************************************************************/
-int n_read(int fd, char *buffer, size_t total)
+int n_recv(int fd, char *buffer, size_t total)
 {
 	register ssize_t rtn;
 	register size_t left = total;
 
 	while (left > 0) {
-		rtn = read(fd, buffer, left);
+		rtn = recv(fd, buffer, left, 0);
```

Comment:
Same comment as below. Why we need this change?

Reply:
I changed this to make the code easier to search: send/recv are more intuitive functions for a send/recv microbenchmark. On my test machine, there was an unexpected 1% performance improvement, too.